rfcs/text/0015-create-file-elf.md

```diff
@@ -86,10 +87,91 @@ Stage 1: Provide a high-level description of example sources of data. This does
 **Stage 2**
 
 ### Real world examples
-
 <!--
 Stage 2: Included a real world example source document. Ideally this example comes from the source(s) identified in stage 1. If not, it should replace them. The goal here is to validate the utility of these field changes in the context of a real world example. Format with the source name as a ### header and the example document in a GitHub code block with json formatting.
+https[://]c4a08b8258a848efb518dcea18a1d11e[.]us-central1[.]gcp[.]cloud[.]es[.]io/goto/22bceecfba7e40710e8c5ddefa853bc8
 -->
+```
+"file": {
+  "elf": {
+    "number_program_headers": 2,
```

Comment:
Thanks for adding these mapping examples. I did note a few fields here not included in the list above.

* `number_program_headers`
* `number_section_headers`
* `packers`

What is being captured with the `number_*` fields? Are those values something that could be retrieved with an agg instead?

Reply:
@dcode can you answer this? 
> What is being captured with the number_* fields? Are those values something that could be retrieved with an agg instead?

Reply:
> Thanks for adding these mapping examples. I did note a few fields here not included in the list above.
> * `packers`

[x] Done

Reply:
Reviewed this. We don't need this. `num_program_headers` refers to the segments and the `num_sections_headers` refers to how many sections are defined in the metadata. Both of these can be computed, if needed, from nested aggregations.

Reply:
Thanks, @dcode.

Can we remove `number_program_headers` and `number_section_headers` from the mapping examples to avoid introducing any confusion?

Reply:
I'll aim for this mañana

Reply:
Done.